src/plugin/cesium/cesiumcamera.js

```diff
@@ -481,6 +481,7 @@ class Camera extends OLCSCamera {
 
   /**
    * @inheritDoc
+   * @param {number=} opt_value The angle, in radians, to rotate by.
```

Comment:
This looks like it might be caused by the issue with the compiler not loading a `requireType` at the appropriate time. If the interface is loaded with `goog.require` does it fix these without the extra JSDoc?

Reply:
Switched both this and `AbstractWebGLSynchronizer` to do a hard require of the interface.